action/protocol/staking/vote_bucket_test.go

```diff
@@ -32,8 +33,8 @@ const (
 
 func newMockKVStore(ctrl *gomock.Controller) db.KVStore {
 	kv := db.NewMockKVStore(ctrl)
-	kmap := make(map[hash.Hash160][]byte)
-	vmap := make(map[hash.Hash160][]byte)
+	kmap := make(map[string]map[hash.Hash160][]byte)
+	vmap := make(map[string]map[hash.Hash160][]byte)
```

Comment:
the test now involved more than 2 namespace
so make the kvmap under map[string]